Source/WebCore/dom/ImageOverlay.cpp

```diff
@@ -346,7 +346,7 @@ static Elements updateSubtree(HTMLElement& element, const TextRecognitionResult&
                     return false;
 
                 for (size_t childIndex = 0; childIndex < childResults.size(); ++childIndex) {
-                    if (childResults[childIndex].text != childTextElements[childIndex]->textContent().stripWhiteSpace())
+                    if (childResults[childIndex].text != childTextElements[childIndex]->textContent().stripLeadingAndTrailingCharacters(deprecatedIsSpaceOrNewline))
```

Comment:
Since the only use here of the result is to compare with another string this could use the `StringView` version for better efficiency.